user/jwt.md

```diff
@@ -0,0 +1,143 @@
+---
+title: JWT addon
+layout: en
+permalink: /user/jwt/
+---
+
+Integration between Travis-CI and third-party services like Sauce Labs relies
+on [encrypted variables](http://docs.travis-ci.com/user/environment-variables/#Encrypted-Variables)
+which works well for trusted branches and committers. For security reasons,
+encrypted variables are not exposed to untrusted pull requests, so third-party
+integrations do not work for pull requests.
+
+The JWT addon replaces encrypted variables with a time-limited authentication
+token, which is exposed to pull requests without security consequences.
+
+For this to work the JWT addon needs to be enabled in the `.travis.yml` file,
+and the third-party need to have integrated with the JWT service and allow
+token-based authentication.
+
+<img src="/user/images/travis_jwt.svg" alt="JWT Travis Flow Diagram">
+
+
+### .travis.yml
+
+Add the encrypted key to the `jwt` section of the `.travis.yml`
+file. For example:
+
+```yml
+addons:
+  jwt:
+     secure: <SAUCE_ACCESS_KEY ENCRYPTED>
+```
+
+You can also configure several services:
+
+```yml
+addons:
+  jwt:
+     saucelabs:
+        secure: <SAUCE_ACCESS_KEY ENCRYPTED>
+     thirdparty:
+        secure: <THIRDPARTY_SHARED_SECRET ENCRYPTED>
+```
+
+### Use the Encrypted Key
+
+The original variable names are available within the Travis CI build as
+environment variables containing the JWT tokens instead of the original values.
+
+For example, using the previous configuration `SAUCE_ACCESS_KEY` and
+`THIRDPARTY_SHARED_SECRET` will be available as environment variables.
```

Comment:
It is perhaps instructive to show what this value would look like to the PR build, and to answer the obvious question, "How secure is this addon?"